packages/tools/pie-wrapper-react/__tests__/wrapper.test.js

```diff
@@ -0,0 +1,23 @@
+import genReactWrapper from '../scripts/react-gen-wrapper.js';
+import removeReactWrapper from '../scripts/react-remove-wrapper.js';
+import fs from 'fs-extra';
+
+const loadJSON = (path) => JSON.parse(fs.readFileSync(__dirname + path));
+
+describe('React Wrapper', () => {
+    it('should generate wrapper from mock custom elements JSON', () => {
+
+      const wrapper = genReactWrapper(loadJSON(`/mocks/mock-custom-elements.json`))
+
+      expect(wrapper).toMatchSnapshot();
+    });
+
+    it('should remove generated wrapper from mock component', () => {
+
+      const wrapper = removeReactWrapper(loadJSON(`/mocks/mock-custom-elements.json`))
+
+      expect(wrapper).toMatchSnapshot();
```

Comment:
Snapshot tests are easy to add, at the same time they can be deceiving.

If you look at the snapshot file there isn't much there: `__tests__/__snapshots__/wrapper.test.js.snap`

Another reason I advise against using snapshot tests instead of regular unit tests is because they don't bring the same advantage: one can use the unit test to understand how an API works without having to read each line of its implementation.

Reply:
So I've added some unit tests as well as the snapshot tests - I think both could be useful but agree more was probably needed to test the logic here 👍